app/services/send_welcome_email_service.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class SendWelcomeEmailService
+  def self.call(current_user:)
```

Comment:
We have pattern for this in `app/lib/service_pattern.rb` that we use in all of the other services. Would be good to be consistent.